src/Network/Network/NetworkInterface/NewAzureNetworkInterfaceCommand.cs

```diff
@@ -56,7 +56,8 @@ public class NewAzureNetworkInterfaceCommand : NetworkInterfaceBaseCmdlet
 
         [Parameter(
             Mandatory = false,
-            ValueFromPipelineByPropertyName = true)]
+            ValueFromPipelineByPropertyName = true,
```

Comment:
This change will also cause change to New-AzNetworkInterface.md. Please regenerate the file.

Reply:
Regenerated the help file